visualization_msgs/MarkerArray objects
geometry_msgs/Point closestFrame
geometry_msgs/Point closestObject
float32 closestDistance
string statusMessage
string statusDescription
